### PR TITLE
PMOAG adds hometown of Chicago

### PR DESCRIPTION
**Pull request by pmoag**

### CODE REVIEW

**Comment by garcia-aa** [comment]
hello

File: PMOAG-Chicago [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Chicago
+### Restaurants
+### Tourist Attractions
+### Local Attractions
+- Lincoln Park Zoo, Lincoln Park
```
Comment (ariestiyansyah) [inline PMOAG-Chicago]: Lincoln Park is a nice place to visit.

Comment (WilliamRClark) [inline PMOAG-Chicago]: @pmoag 

Any chickens at the zoo?  :chicken:  :baby_chick:

Comment (pmoag) [inline PMOAG-Chicago]: Yes, they have a "farm" in the middle of the Zoo, William!

**Pull request merged**